<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="浪里个浪" ignored="1" name="examplel" tests="3" failures="1" timestamp="29 七月 2022 08:32:27 GMT" time="1.386" errors="0">
  <testcase name="wo404" time="0.878" classname="com.hxk.tesngDemo.courseRecord404">
    <failure type="java.lang.AssertionError" message="服务器返回的状态码不是200 expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: 服务器返回的状态码不是200 expected [200] but found [404]
at com.hxk.tesngDemo.courseRecord404.wo404(courseRecord404.java:31)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- wo404 -->
  <testcase name="test1" time="0.002" classname="com.hxk.tesngDemo.TestDemo"/>
  <testcase name="index" time="0.353" classname="com.hxk.tesngDemo.index"/>
  <testcase name="test2" time="0.0" classname="com.hxk.tesngDemo.TestDemo2">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- examplel -->
